pkg/graph/resolvers/trb_request.go

```diff
@@ -222,6 +222,87 @@ func UpdateTRBRequestTRBLead(
 	return updatedTrb, err
 }
 
+// CloseTRBRequest closes a TRB request and sends an email if recipients are specified
+func CloseTRBRequest(
+	ctx context.Context,
+	store *storage.Store,
+	emailClient *email.Client,
+	fetchUserInfo func(context.Context, string) (*models.UserInfo, error),
+	fetchUserInfos func(context.Context, []string) ([]*models.UserInfo, error),
+	id uuid.UUID,
+	reasonClosed string,
+	notifyEUAIDs []string,
+) (*models.TRBRequest, error) {
+	// Query the TRB request, attendees in parallel
+	errGroup := new(errgroup.Group)
+
+	// Query the TRB request
+	var trb *models.TRBRequest
+	var errTRB error
+	errGroup.Go(func() error {
+		trb, errTRB = store.GetTRBRequestByID(ctx, id)
+		return errTRB
+	})
+
+	if errG := errGroup.Wait(); errG != nil {
+		return nil, errG
+	}
+
+	// Check if request is already closed so an unnecesary email won't be sent
+	if trb.Status != models.TRBSOpen {
+		return nil, errors.New("Cannot close a TRB request that is not open")
+	}
+
+	trbChanges := map[string]interface{}{
+		"status": models.TRBSClosed,
+	}
+
+	err := ApplyChangesAndMetaData(trbChanges, trb, appcontext.Principal(ctx))
+	if err != nil {
+		return nil, err
+	}
+
+	updatedTrb, err := store.UpdateTRBRequest(ctx, trb)
+	if err != nil {
+		return nil, err
+	}
+
+	requester, err := fetchUserInfo(ctx, trb.CreatedBy)
+	if err != nil {
+		return nil, err
+	}
+
+	notifyUserInfos, err := fetchUserInfos(ctx, notifyEUAIDs)
+	if err != nil {
+		return nil, err
+	}
+
+	recipientEmails := make([]models.EmailAddress, 0, len(notifyUserInfos)+1)
```

Comment:
If using `make`, I think you can just specify it like
```suggestion
	recipientEmails := make([]models.EmailAddress, len(notifyUserInfos)+1)
```

and then later, instead of using `append()`, set each item at the index.


Alternatively, if you'd rather use `append`, you can probably just do 
```suggestion
	recipientEmails := []models.EmailAddress{}
```